README.md

```
@@ -10,6 +232,34 @@ It evolves incrementally, based on actual use and experience.
 
 ---
 
+## Git Hooks
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)